azerothcore/azerothcore-wotlk
# fix(DB/quest): Add 2 day time delay to Gubber Blump's letter

## Changes Proposed:
- Adds a 2 day delay to the reward letter Gubber Blump sends you after completing [The Family and the Fishing Pole](https://tbc.wowhead.com/quest=1141/the-family-and-the-fishing-pole). At the moment he sends the mail instantly after completing the quest, which is not correct. The 2 day delay is a best guess from sources.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7982
- Closes https://github.com/chromiecraft/chromiecraft/issues/1769

## SOURCE:

"Two days after doing this quest I received mail from Mr. Blump. He thanked me for helping with the quest, and sent along a fishing bauble with the letter."

"I did this quest days ago (maybe over a week? *shrug*), so was quite surprised when I got mail from Gubber! "

"a day or so later I received a letter from him thanking me with an Aquadynamic Fish Attractor. "

https://tbc.wowhead.com/quest=1141/the-family-and-the-fishing-pole

## Tests Performed:

- ran SQL, no errors/warnings, 1 row changed as expected.
- Checked in game - ported to Gubber, did quest, no mail appeared. Logged out, set server clock forwards 2 days, and waited, and the mail appeared in my mailbox right on time.

## How to Test the Changes:


1. .go c 39062
2. Accept quest
3. .additem [darkshore grouper] 6
4. Handin.
5. Note no mail arrives.
6. Logout and set server clock forwards 2 days.
7. Check mailbox again.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
